Utilities/DataFlow/include/DataFlow/PayloadMerger.h

```diff
@@ -0,0 +1,123 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
```

Comment:
At least for the new files we should start adding the correct public link with http instead https.